README.md

```diff
@@ -166,7 +166,9 @@ name-service         1/1     107m
 ## Apache RocketMQ Community
 * [RocketMQ Streams](https://github.com/apache/rocketmq-streams): A lightweight stream computing engine based on Apache RocketMQ.
 * [RocketMQ Flink](https://github.com/apache/rocketmq-flink): The Apache RocketMQ connector of Apache Flink that supports source and sink connector in data stream and Table.
-* RocketMQ Muli-Language Clients
+* [RocketMQ APIs](https://github.com/apache/rocketmq-apis): RocketMQ ProtoBuf Protocol
+* [RocketMQ Clients](https://github.com/apache/rocketmq-clients): gRPC/ProtoBuf-based RocketMQ Clients
+* Remoting-based RocketMQ Clients
```

Comment:
IMO, it would be better to name “RocketMQ Native Protocol Clients” instead of “Remoting-based RocketMQ Clients”

Reply:
How about slightly rephrase to "RocketMQ Native Protocol-Based Clients"? May sounds more formal.

Reply:
Nice suggestion